js-setup/js/component-initializer.js

```diff
@@ -0,0 +1,153 @@
+import Loader from './loader';
+import ads from '../../ads';
+import tracking from '../../tracking';
+import date from '../../date';
+import header from '../../header';
+import promoMessages from '../../promo-messages';
+import cookieMessage from '../../cookie-message';
+import welcomeMessage from '../../welcome-message';
+import messagePrompts from '../../message-prompts';
+import footer from '../../footer';
+import myft from '../../myft';
+
+export const presets = {
+	discrete: {
+		header: true,
+		footer: true,
+		date: true,
+		welcomeMessage: true
+	},
+	complete: {
+		header: true,
+		footer: true,
+		date: true,
+		cookieMessage: true,
+		welcomeMessage: true,
+		myft: true,
+		messagePrompts: true,
+		promoMessages: false,
+		ads: true
+	}
+};
```

Comment:
Perhaps in another PR reconciling presets, _entry and main.js can be done. I agree that it's lacking a sense of there being a single source of truth